### ISSUE TITLE
re-use new gradient color in all background related controls

### ISSUE THREAD

**Issue by selul** [Codeinwp/otter-blocks#180]
The new gradient control from the core seems more solid than the current one that we use. 

We can look into this and see re-use the same control available in the cover block 
https://share.getcloudapp.com/YEuA2xOd

**Comment by HardeepAsrani** [comment]
@selul Yes, it's pretty good. The only issue is that it's only available in version 5.4. What do you think?

**Comment by selul** [comment]
we can have this in mind for future releases, maybe after a few months. We can't use the package standalone for older wp versions?
